src/mounts.cpp

```diff
@@ -65,7 +63,7 @@ Mount* Mounts::getMountByID(uint8_t id)
 
 Mount* Mounts::getMountByName(const std::string& name) {
 	auto it = std::find_if(mounts.begin(), mounts.end(), [&name](const Mount& mount) {
-		return boost::iequals(name, mount.name);
+		return strcasecmp(name.c_str(), mount.name.c_str());
```

Comment:
`strcasecmp` returns 0 if strings are equal, so you need to use `return strcasecmp(...) == 0`.

Yes, glibc sucks a lot.